dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -55,6 +61,7 @@ import { updateTabName } from './slices/query_tab_slice';
 import { selectCountDistribution } from './slices/count_distribution_slice';
 import { selectExplorerVisualization } from './slices/visualization_slice';
 import { IExplorerProps } from '../../../common/types/explorer';
+import { threadId } from 'worker_threads';
```

Comment:
what is this?

Reply:
oh this is never used anywhere, i'll remove it